lib/tasks/projects.rake

```diff
@@ -113,4 +113,11 @@ namespace :projects do
       end
     end
   end
+
+  desc 'Refresh dependent repos materialized view'
+  task refresh_dependent_repos_view: :environment do
+    # This task can take about 30 minutes,
+    exit if ENV['READ_ONLY'].present?
+    ActiveRecord::Base.connection.execute("refresh materialized view project_dependent_repositories")
```

Comment:
this should probably be done with `CONCURRENTLY` so that reads can continue as the refresh happens